src/pytest_cases/case_parametrizer_new.py

```diff
@@ -265,9 +266,18 @@ def get_all_cases(parametrization_target,  # type: Callable
 
         filters += (filter,)
 
+    # Validate that we have a parametrization target if required for retrieving cases
+    if parametrize_with_cases is None:
```

Comment:
why do you refer to `parametrize_with_cases` here ?

Reply:
Fixed